app/tests/unit/components/validators/validators.customValidators.spec.js

```diff
@@ -51,6 +51,7 @@ describe('customValidators.docGen', () => {
       template: {
         content: 'ZHNmc2Rmc2RmZHNmc2Rmc2Rmc2Rm',
         contentFileType: 'docx',
+        outputFileType: 'docx',
```

Comment:
Was outputFileType added to get this existing test to pass? Because outputFileType is not a required field, so this test is meant to pass without any optional fields. If the test was failing then I think it's probably evidence that the validator is not behaving as intended. Is the validator possibly requiring an outputFileType even though it's an optional field?

Reply:
yes. i added outputFileType as a way to get that test to pass. 
I'll take a look a fixing the validator now